source/endpoint-reference/endpoint-variable.rst

```diff
@@ -401,6 +401,53 @@ Behaves sames as PATCH.
 
 .. _endpoint-variables-weights:
 
+
+Personal Variable Order Catalog
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``/datasets/{id}/variables/personal/variables/``
+
+The personal variable order comes accompanied with a catalog containing
+a variables present in the order's graph. This should provide access to
+metadata of those variables.
```

Comment:
Perhaps replace:

- `This should provide access to metadata of those variables.`

with:

- `This provides access to the metadata for those variables.`

Reply:
Done